R/plots.R

```diff
@@ -522,7 +523,7 @@ dotPlot <- function (markers,
                      scale.by = "radius",
                      scale.min = NA,
                      scale.max = NA,
-                     verbose=TRUE,
+                     verbose=FALSE,
```

Comment:
Why was this changed?

Reply:
Honestly, I introduced the progress bar at a time where I was in love with these things. However, in my experience this function is quite fast, no need to show bars per default